src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService checkUser = new UserService();
```

Comment:
I would recommend to make `UserService` a class member, as it will improve flexibility and simplify testing. Also, consider more appropriate variable name for the service, like `userService`.